package.json

```diff
@@ -95,7 +95,8 @@
     "fast-xml-parser": "4.2.5",
     "semver": "7.5.2",
     "tough-cookie": "4.1.3",
-    "aws-cdk-lib": "2.80.0"
+    "aws-cdk-lib": "2.80.0",
+    "prismjs": "^1.27.0"
```

Comment:
do we need the axios upgrade from last night?

Reply:
No, everything is using the upgraded axios in next-release/main